maps_dashboards/server/saved_objects/map_saved_object.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from 'opensearch-dashboards/server';
+
+export const mapSavedObjectsType: SavedObjectsType = {
+  name: 'map',
+  hidden: false,
+  namespaceType: 'agnostic',
```

Comment:
how did you determine this value?

Reply:
Currently I just keep it as `agnostic ` which is a global type, will need to figure out which namespaceType we really need.